src/context.h

```diff
@@ -140,6 +143,8 @@ class PopulateCtx : public GenCtx {
     size_t arith_depth;
     // If the test will actually execute the code
     bool taken;
+
+    bool inside_omp_simd;
```

Comment:
Would be good to have comment here. Is pragma OMP SIMD attached to this region directly or it can be nested region?

Reply:
Done